assets/stylesheets/comments.scss

```diff
@@ -11,25 +25,6 @@
         color: #000;
     }
 
-    .comment-buttons {
-        margin-top: -10px;
-        margin-bottom: 15px;
-    }
-
```

Comment:
Note that this class doesn't seem to be use anywhere and the comment buttons still look good without it. So the removal is intentional.